tests/integration/test_data_cli.py

```diff
@@ -779,12 +785,192 @@ def test_search_update_fail(invoke, search_id, search_filter):
+        assert result.exit_code == 0
+
+        path = Path(folder, 'img.tif')
+
+        assert path.name == 'img.tif'
+        assert path.is_file()
+
+        if exists and not overwrite:
+            assert path.read_text() == 'i exist'
+            assert len(result.output) == 0
+        else:
+            assert len(path.read_bytes()) == 527
+            assert path.name in result.output
+
+
+@respx.mock
+def test_asset_activate(invoke):
+    mock_resp = httpx.Response(HTTPStatus.OK)
+    dl_url = f'{TEST_URL}/1?token=IAmAToken'
+    respx.get(dl_url).return_value = mock_resp
+
+    basic_udm2_asset = {
+        "_links": {
+            "_self": "SELFURL",
+            "activate": "ACTIVATEURL",
+            "type": "https://api.planet.com/data/v1/asset-types/basic_udm2"
+        },
+        "_permissions": ["download"],
+        "md5_digest": None,
+        "status": 'active',
+        "location": dl_url,
+        "type": "basic_udm2"
+    }
+
+    runner = CliRunner()
+    result = invoke(['asset-activate', json.dumps(basic_udm2_asset)],
+                    runner=runner)
+
+    assert not result.exception
+
+
+@respx.mock
+def test_asset_wait(invoke):
+    mock_resp = httpx.Response(HTTPStatus.OK)
+    dl_url = f'{TEST_URL}/1?token=IAmAToken'
+    respx.get(dl_url).return_value = mock_resp
+
+    basic_udm2_asset = {
+        "_links": {
+            "_self": "SELFURL",
+            "activate": "ACTIVATEURL",
+            "type": "https://api.planet.com/data/v1/asset-types/basic_udm2"
+        },
+        "_permissions": ["download"],
+        "md5_digest": None,
+        "status": 'active',
+        "location": dl_url,
+        "type": "basic_udm2"
+    }
+
+    runner = CliRunner()
+    result = invoke(
+        ['asset-wait', json.dumps(basic_udm2_asset), '--delay', '0'],
+        runner=runner)
+
+    assert not result.exception
+    assert "state: active" in result.output
+
+
+@respx.mock
+def test_asset_get(invoke):
+    item_type = 'PSScene'
+    item_id = '20221003_002705_38_2461xx'
+    asset_type_id = 'basic_udm2'
+    dl_url = f'{TEST_URL}/1?token=IAmAToken'
+
+    basic_udm2_asset = {
+        "_links": {
+            "_self": "SELFURL",
+            "activate": "ACTIVATEURL",
+            "type": "https://api.planet.com/data/v1/asset-types/basic_udm2"
+        },
+        "_permissions": ["download"],
+        "md5_digest": None,
+        "status": 'active',
+        "location": dl_url,
+        "type": "basic_udm2"
+    }
+
+    page_response = {
+        "basic_analytic_4b": {
+            "_links": {
+                "_self":
+                "SELFURL",
+                "activate":
+                "ACTIVATEURL",
+                "type":
+                "https://api.planet.com/data/v1/asset-types/basic_analytic_4b"
+            },
+            "_permissions": ["download"],
+            "md5_digest": None,
+            "status": "inactive",
+            "type": "basic_analytic_4b"
+        },
```

Comment:
I'm curious why this part was included, I don't see where it's being used by the test.

Reply:
`page_response` is being used by `mock_resp` on line 958. i.e., 
`mock_resp = httpx.Response(HTTPStatus.OK, json=page_response)`